### DISCUSSION TITLE
Struggling with link

### DISCUSSION THREAD

**Discussion by Berto979** [noraj/OSCP-Exam-Report-Template-Markdown#54] [Q&A]
HI, 
yesterday i'm exported a page from Notion in markdown and i want to use this repository to create my report.
So i execute this command `sudo ruby osert.rb generate -i /home/kali/OSCP-Exam-Report-Template-Markdown/src/oscp_t_4.md -o /home/kali` but it raise an error during pdf generation.

```
Error producing PDF.
! Argument of \MT@res@a has an extra }.
<inserted text> 
                \par 
l.836 ...\textquotesingle{}s\%20test/test/test.md}
```

After i few hours i checked that this error is raised from a ' (apostrophe), infact when i delete it report was generated perfectly.
Link that created a lot of problem (in order not to violate the exam rules I created an example link)
**Proof of Concept Code:**

```md
[test/test.md at GitHub](https://github.com/test/test/test/test/test/test's%20test/test/test.md).
```

Could anyone tell me how I could solve this problem by not changing the link?

**Comment by noraj** [comment]
It sounds like a LaTeX limitation, it would be best just to URL-encore the quote in the URL.
